features/users/new_external_user.feature

```diff
@@ -22,7 +22,7 @@ Feature: Create new external user
     Given I should not see 'VAT registered'
     And I should not see 'Supplier number'
     When I click govuk checkbox 'Advocate'
-    And the page should be accessible
+    And the page should be accessible skipping 'aria-allowed-attr'
```

Comment:
**question:** not related to this PR directly but do we need this many accessibility checks. Will one per page not be sufficient? or is it checking only visible elements each time

Reply:
@jsugarman Yes, as the feature changes the dom, we need to make sure that all changes are checked for accessibility.